src/ResourceManager/Sql/Commands.Sql.Test/ScenarioTests/ManagedInstanceKeyVaultKeyTests.ps1

```diff
@@ -0,0 +1,162 @@
+﻿# ----------------------------------------------------------------------------------
+#
+# Copyright Microsoft Corporation
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ----------------------------------------------------------------------------------
+		
+$mangedInstanceRg = "MlAndzic_RG"
+$managedInstanceName = "midemoinstancebc"
+$keyVaultName = "mitest-doNotDelete"
+$keyName = "mitest-key"
+$keyId = "https://mitest-donotdelete.vault.azure.net/keys/mitest-key/e97043455e14493ca05c3642549aedba"
+$keyVersion = "e97043455e14493ca05c3642549aedba"
+$tdeKeyName = $keyVaultName + "_" + $keyName + "_" + $keyVersion
```

Comment:
We need instructions for this setup before merging this PR.

Reply:
Added this to ps1 file as well
# A managed instance can be provisioned using instructions here https://docs.microsoft.com/en-us/azure/sql-database/sql-database-managed-instance-get-started
# currently this takes about 2-3 hours